src/manageiq_client/api.py

```diff
@@ -230,6 +240,13 @@ def reload_if_needed(self):
         if self._data is None:
             self.reload()
 
+    def query_string(self, **params):
+        """Specify query string to use with the collection.
```

Comment:
I was also unsure how to call it, but since "query string" is well known term, I went with it.